tests/unit/config/test_qtargs.py

```diff
@@ -154,6 +154,36 @@ def test_in_process_stack_traces(self, monkeypatch, parser, backend, version_pat
             assert '--disable-in-process-stack-traces' in args
             assert '--enable-in-process-stack-traces' not in args
 
+    @pytest.mark.parametrize(
+        'qt_version, qt6, value, has_arg',
+        [
+            ('5.15.2', False, 'auto', False),
+            ('6.5.3', True, 'auto', True),
+            ('6.6.0', True, 'auto', False),
+            ('6.5.3', True, 'always', True),
+            ('6.5.3', True, 'never', False),
+            ('6.6.0', True, 'always', True),
+        ],
+    )
+    def test_accelerated_2d_canvas(
+        self,
+        parser,
+        version_patcher,
+        config_stub,
+        monkeypatch,
+        qt_version,
+        qt6,
+        value,
+        has_arg,
+    ):
+        version_patcher(qt_version)
+        config_stub.val.qt.workarounds.disable_accelerated_2d_canvas = value
+        monkeypatch.setattr(machinery, 'IS_QT6', qt6)
```

Comment:
Could use `qt_version.startswith("6.")` instead. Then again, tests should avoid being too smart, so this is fine in my book too.